README.md

```diff
@@ -0,0 +1,34 @@
+<div align="center">
+  <img src="./brand/dragonfly-rainbow.svg" width="300">
+</div>
+
+# Hypothesis
+
+* [Website](https://hypothesis.works/)
+* [Documentation](https://hypothesis.readthedocs.io/en/latest/)
+* [Source code](https://github.com/hypothesisWorks/hypothesis/)
+* [Contributing](https://github.com/HypothesisWorks/hypothesis/blob/master/CONTRIBUTING.rst)
+* [Community](https://hypothesis.readthedocs.io/en/latest/community.html)
+
+Hypothesis is the canonical property-based testing library for Python. A property-based test asserts something for *all* inputs, and lets Hypothesis generate them — including inputs you may not have thought of.
```

Comment:
"canonical" is maybe too strong/grandiose a word here. I considered "definitive" too, but that seems even worse.

Reply:
"premier" maybe?

Reply:
makes it sound too new and unstable, I want to give off more "mature and used" vibes

Reply:
"leading"

Sorry, deep in bike shedding territory now :-) I think all the suggestions are fine.

Reply:
Actually I think this is not bike shedding at all! Ever word on the readme is super important because it's a reader's first impression of us, so I'm happy to quibble over details. (my favorite is still canonical - there really *aren't* any other pbt libraries in python)

Reply:
I'd be inclined to drop the adjective, and just say "Hypothesis is the property-based testing library for Python"

Reply:
that works for me